### PR TITLE
Add authentication documentation

### PR DESCRIPTION
**Pull request by datamel**
Adding authentication information to the docs.

### CODE REVIEW

**Comment by datamel** [comment]
PR #116 adds a security section to the docs, I have added basic user information in this PR - mainly detailing what users should expect in terms of file structure. I can add a more detailed documentation update once the security section has been added in #116.

**Comment by datamel** [comment]
Partially addresses #123

File: src/user-guide/running-suites.rst [hunk 1]
```diff
@@ -248,6 +248,44 @@ commands can read this file, if they have access to it, to find the target
 :term:`scheduler`.
 
 
+.. _Authentication Files:
+
+Authentication Files
+--------------------
+
+Cylc uses CurveZMQ to ensure that any data, sent between the
```
Comment (oliver-sanders) [inline src/user-guide/running-suites.rst:256]: Perhaps link to curvezmq.

**Review by oliver-sanders — APPROVED** [review]

**Comment by oliver-sanders** [comment]
> Partially addresses #123

Could you make a note of anything that still requires documentation on that issue.

File: src/user-guide/running-suites.rst (hunk 1 above)
Comment (datamel) [reply src/user-guide/running-suites.rst:256]: Good plan.

**Comment by datamel** [comment]
> > Partially addresses #123
> 
> Could you make a note of anything that still requires documentation on that issue.

No probs, I'll get that sorted.

**Review by hjoliver — APPROVED** [review]

**Comment by hjoliver** [comment]
That's good, thanks @datamel

**Pull request merged**